clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -36,7 +36,7 @@ def api_call_latency(title, verb, scope, threshold):
 CLUSTERLOADER_PANELS = [
```

Comment:
Actually, let's get rid of this whole panel. Without being able to plot requests coming from clusterloader there is no point in having it.

Reply:
Removed and `CLUSTERLOADER_PANELS ` and `api_call_latency` definition that is not used anymore.